### ISSUE TITLE
[QST] to_csv and other methods fail due to lack of upstream implementation

### ISSUE THREAD

**Issue by beckernick** [rapidsai/dask-cudf#127]
## Description
As a user, I want to write my dask_cudf dataframes to CSV files for storage. Currently, a `to_csv` API is exposed but causes an error due to a lack of upstream `to_csv` implementation in cuDF. This is a known feature request for cuDF, but is relevant here as the exposed `to_csv` API may confuse users of dask_cudf.

The broader question is, how do we want to handle broken dask_cudf API functionality due to lack of upstream cuDF support? Do we want to take any proactive action to protect/warn users?

@mrocklin @randerzander  @quasiben

**Comment by mrocklin** [comment]
I agree that this would be super useful.  We could add implementations for every unsupported function onto dask-cudf?  Maybe there is some way of doing this programmatically.  I don't know of a good solution off-hand, but if someone has an idea and wants to implement it I say go for it.

In the mean time, here are some things that are and are not supported: https://rapidsai.github.io/projects/cudf/en/latest/dask-cudf.html

**Comment by randerzander** [comment]
to_csv in particular may be resolved when rapidsai/cudf/pull/1542 merges
